HelpSource/Classes/Pclump.schelp

```diff
@@ -0,0 +1,38 @@
+TITLE:: Pclump
+summary:: Clumps the pattern into groups of size n.
```

Comment:
"of size n" is meaningless here since you haven't defined `n`. a more appropriate wording might be "Clumps the pattern into groups of user-defined sizes."

Reply:
Yes that's better.